### PR TITLE
Attempt to increase page_size limit for sync

### PR DESCRIPTION
**Pull request by rochacbruno**
[noissue]

### CODE REVIEW

File: pulp_ansible/app/tasks/collections.py [hunk 1]
```diff
@@ -385,6 +384,7 @@ def __init__(self, remote, repository, optimize):
         self._unpaginated_collection_metadata = None
         self._unpaginated_collection_version_metadata = None
         self.last_synced_metadata_time = None
+        self.page_size = settings.get("SYNC_PAGE_SIZE", 1000)
```
Comment (gerrod3) [inline pulp_ansible/app/tasks/collections.py:387]: Do you set this on galaxy's side?

**Review by gerrod3 — APPROVED** [review]
I think this PR should be used over #558. It's less invasive and is less likely to timeout. While it's not as performant I think we need more time to find a long term solution then trying to ship a last-minute temporary solution.

File: pulp_ansible/app/galaxy/v3/pagination.py [hunk 2]
```diff
@@ -10,7 +10,7 @@ class LimitOffsetPagination(pagination.LimitOffsetPagination):
     """
 
     default_limit = 10
-    max_limit = 100
+    max_limit = 1000
```
Comment (alikins) [inline pulp_ansible/app/galaxy/v3/pagination.py:13]: Does this imply that OptimizedLimitOffsetPagination won't be used now? If so, presumably https://github.com/ansible/galaxy_ng/pull/726/files#diff-e43be673099bbec9ffef097ed5503589ff4597e387c89ed3ea2fdf9365e3d05dR81 needs updating as well

File: pulp_ansible/app/tasks/collections.py (hunk 1 above)
Comment (rochacbruno) [reply pulp_ansible/app/tasks/collections.py:387]: it defaults to 1000 but can be set to other value via PULP_SYNC_PAGE_SIZE variable

File: pulp_ansible/app/galaxy/v3/pagination.py (hunk 2 above)
Comment (rochacbruno) [reply pulp_ansible/app/galaxy/v3/pagination.py:13]: if this PR is accepted then 726 can be closed without merging.

**Comment by rochacbruno** [comment]
@gerrod3 lets close it and go with #562 ?

**Pull request closed without merging**